src/names/applications.cpp

```diff
@@ -175,3 +179,11 @@ DescFromName (const valtype& name, NameNamespace ns)
         }
     }
 }
+
+bool
+IsValidJSON (const std::string& text){
+    UniValue v;
+
+    LogDebug(BCLog::VALIDATION, "ValidJSON function triggered");
```

Comment:
This log message was useful during testing but I think we can remove it now that the code is tested to work. Otherwise it'll spam logs a lot (I guess one message per keystroke in this textbox?).